### PR TITLE
[sysconfig] Remove default value for JAVA_HOME

### PR DESCRIPTION
**Pull request by arthurzenika**
This breaks with deployement of version 7.5 of ES

### PR progress checklist (to be filled in by reviewers)

- [ ] Changes to documentation are appropriate (or tick if not required)
- [ ] Changes to tests are appropriate (or tick if not required)
- [ ] Reviews completed

---

#### Primary type

- [ ] `[build]`    Changes related to the build system
- [ ] `[chore]`    Changes to the build process or auxiliary tools and libraries such as documentation generation
- [ ] `[ci]`       Changes to the continuous integration configuration
- [ ] `[feat]`     A new feature
- [ ] `[fix]`      A bug fix
- [ ] `[perf]`     A code change that improves performance
- [ ] `[refactor]` A code change that neither fixes a bug nor adds a feature
- [ ] `[revert]`   A change used to revert a previous commit
- [ ] `[style]`    Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc.)

#### Secondary type

- [ ] `[docs]`     Documentation changes
- [ ] `[test]`     Adding missing or correcting existing tests

### Does this PR introduce a `BREAKING CHANGE`?

No.

### Documentation checklist

- [ ] Updated the `README` (e.g. `Available states`).
- [ ] Updated `pillar.example`.

### Testing checklist

- [ ] Included in Kitchen (i.e. under `state_top`).
- [ ] Covered by new/existing tests (e.g. InSpec, Serverspec, etc.).
- [ ] Updated the relevant test pillar.

### CODE REVIEW

**Comment by arthurlogilab** [comment]
Does anyone have a minute to take a look at this ? Thanks in advance.

File: elasticsearch/files/sysconfig [hunk 1]
```diff
@@ -1,6 +1,3 @@
-{%- if 'JAVA_HOME' not in sysconfig.keys() %}
-JAVA_HOME=/usr/lib/java
```
Comment (aboe76) [inline elasticsearch/files/sysconfig]: Why default to /usr/lib/java?

Comment (arthurlogilab) [reply elasticsearch/files/sysconfig]: This was the old default I guess. This merge request is removing that. 

Thanks for the review by the way !

**Review by aboe76 — APPROVED** [review]

**Comment by myii** [comment]
Merged, thanks for the patience, @arthurlogilab!

**Pull request merged**